src/Components/CollectionsHubRails/ArtistSeriesRail/index.tsx

```diff
@@ -14,33 +15,49 @@ export const ArtistSeriesRail: React.FC<ArtistSeriesRailProps> = ({
   collectionGroup,
 }) => {
   const { members } = collectionGroup
+  const broswerWidth = useWindowSize()
+
   return (
     <Content mt={2} py={3}>
       <Serif size="5" mb={1}>
         Trending Artist Series
       </Serif>
       <Carousel
-        height="200px"
-        width="500px"
+        height="250px"
         options={{
-          groupCells: sd.IS_MOBILE ? 1 : 4,
+          groupCells: sd.IS_MOBILE ? 1 : broswerWidth > 1024 ? 4 : 3,
```

Comment:
Nice 👍

Reply:
@damassi How about this ?
groupCells: (() => {
            if (sd.IS_MOBILE) {
              return 1
            } else {
              if (broswerWidth > 1024) {
                return 4
              } else {
                return 3
              }
            }
          })(),

Reply:
Rather than writing a self-invoking function, it would be better to keep things minimal and assign to a variable just above, within the render method and before the `return`.

Reply:
@damassi How about this time?
let groupCellsValue: number
  if(sd.IS_MOBILE){
    groupCellsValue = 1
  }else{
    if(broswerWidth > 1024 ){
      groupCellsValue =4
    }
    else
    groupCellsValue =3
  }
......
groupCells : groupCellsValue,

Reply:
@jiahaodavid - Would you mind updating these comments to support proper github code snippet formatting?

Reply:
`groupCellsValue` -> `groupCells`,  the same as the setting. That way things are 1:1, and we don't have to worry about variables diverging through time, and if we ever needed to update something globally, all relevant references would be returned. (Also, in general, no need to append "value" to a variable as by it's nature it's already a value.)

Reply:
Yes. ```let groupCellsValue: number
if(sd.IS_MOBILE){
groupCellsValue = 1
}else{
if(broswerWidth > 1024 ){
groupCellsValue =4
}
else
groupCellsValue =3
}```

Reply:
Yes. I made some change. How about this? If ok, I will apply to other condition checks like this one.
```
let groupCells: number
  if(sd.IS_MOBILE){
  groupCells = 1
  }
  else{
      if(broswerWidth > 1024 ){
      groupCells =4
      }
      else
      groupCells =3
  }

Reply:
Check out:  https://help.github.com/en/articles/creating-and-highlighting-code-blocks for better highlights.

But yeah, logic looks good!

Reply:
@damassi Hi, I make some change. You can see below. I use the better highlight this time. Last time I manually added the space to the code. 😭  
```typescript
let groupCells: number
  if (sd.IS_MOBILE) {
    groupCells = 1
  } else {
    if (broswerWidth > 1024) {
      groupCells = 4
    } else groupCells = 3
  }
```
..... 
```typescript
options={{
          groupCells,
          wrapAround: sd.IS_MOBILE ? true : false,
          cellAlign: "left",
          pageDots: false,
          draggable: sd.IS_MOBILE ? true : false,
          contain: true,
        }}
```
Also, Christina gave me some feedback. I made some change on other condition check. Thx @xtina-starr 
```typescript
const isSmallerViewpoint = broswerWidth < 1024
```
...
```typescript 
renderRightArrow={({ Arrow }) => {
          return (
            <ArrowContainer>
              {members.length > 4 ? (
                <Arrow />
              ) : (
                <Arrow showArrow={members.length === 4 && isSmallerViewpoint} />
              )}
            </ArrowContainer>
          )
        }
```

Reply:
It seems I have to make the condition check before rendering the **Arrow** components. Otherwise, I will fail the test. Because the test only checks the length of the <ArrowButton> inside the <Arrow>. So, once the <Arrow> got rendered, the arrows maybe not show in the browser but the test still can detect 2 **Arrowbutton**. So, I am going to make some change like this. You can see below. And what do you think? @damassi @xtina-starr 
```typescript
renderRightArrow={({ Arrow }) => {
          const smallerViewpointAndFourItems =
            isSmallerViewpoint && members.length === 4
          return (
            <ArrowContainer>
              {members.length > 4 ? (
                <Arrow />
              ) : (
                smallerViewpointAndFourItems && <Arrow showArrow={true} />
              )}
            </ArrowContainer>
          )
        }
```